docs/notebooks/des-gaia.ipynb

```diff
@@ -0,0 +1,660 @@
+{
```

Comment:
<p>Explain why it has higher order. This is because more objects -&gt; need to portion more finely.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/astronomy-commons/lsdb/pull/166/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>

Reply:
<p>Added something to address it</p><p> </p>